argilla/docs/how_to_guides/custom_fields.md

```diff
@@ -0,0 +1,184 @@
+---
+description: Learn how to create custom fields using HTML, CSS, and JavaScript templates in Argilla.
+---
+
+# Custom fields with JavaScript, HTML and CSS templates
```

Comment:
The title is slightly misleading because the user doesn't need all of this to make a template. Personally, i think something functional would attract more users to it. 

```suggestion
# Custom fields with layout templates
```

That said, maybe there's appeal in the languages to web developers.